### DISCUSSION TITLE
Creating subtable from a document field

### DISCUSSION THREAD

**Discussion by BeeBombshell** [AntlerVC/firetable#956] [Support / Q&A] [answered]
> This discussion is in reference to a Discord conversation.

I want to create a Sub-table for cards but my cards are not a subcollection.
The card list is a field of the deck document. 

How should i go about it?

<img width="547" alt="image" src="https://user-images.githubusercontent.com/53828745/202520232-675334a2-8d99-4852-8a87-d9d007abde36.png">

**Comment by harinij** [comment, marked as answer]
This requires a new column type to support a sub-table for arrays with table view instead of sub-collection. An issue for this is created here: #959
